### PR TITLE
Philippines: National Capital Region updates

### LINKED ISSUE #1214: [Status: Ready for Merge] Philippines: Erroneous hierarchy and labels in National Capital Region
- [City of Makati](https://spelunker.whosonfirst.org/id/1108697351/)
  - Description: This is encoded as a child of another city, the region entity labeled [Quezon City](https://spelunker.whosonfirst.org/id/1108697351/). On the other hand, its geometry is actually _bigger_ than Quezon City, and corresponds to what is locally known as the National Capital Region, which isn't reflected anywhere in the hierarchy. There's a separate [entity](https://spelunker.whosonfirst.org/id/85676413/), a region simply called Makati, that contains a shape actually corresponds to the actual city's administrative boundaries.
  - Suggestions:
    - Rename to National Capital Region
    - Change placetype to `region`
    - Set parent to [Philippines](https://spelunker.whosonfirst.org/id/85632509/)
- Cities within the National Capital Region
  - Description: 
    - There are entities that refer to actual cities within the National Capital Region (e.g. [Quezon City](https://spelunker.whosonfirst.org/id/85676395/), [Marikina](https://spelunker.whosonfirst.org/id/85676399/), [Pasay](https://spelunker.whosonfirst.org/id/85676415/)), but they're designated as `regions` and direct children of the Philippines country entity.
    - There are separate entities (e.g. [Marikina](https://spelunker.whosonfirst.org/id/421200223/), [Pasay](https://spelunker.whosonfirst.org/id/1141907273/)) that are designated as `localities` that seem to be duplicates, without any shapes.
    - These are labeled "National Capital Region" according to their `eng_x_preferred` values.
    - The above leads to some bizarre hierarchies: 
image
  - Suggestions:
    - Change these entities' placetypes from  `region` to `locality`, as per the [placetype guidelines](https://github.com/whosonfirst/whosonfirst-placetypes#locality).
    - Change their parents to [this](https://spelunker.whosonfirst.org/id/1108697351/) which seems to correspond to the National Capital Region.
    - Less a suggestion and more a question: it also seems appropriate to deduplicate and merge the data for the existing localities (so, say, merge [Marikina](https://spelunker.whosonfirst.org/id/85676399/) and [Marikina](https://spelunker.whosonfirst.org/id/421200223/)). Is there any accepted best practice or protocol for doing so?

### PR DESCRIPTION
**Pull request by stepps00**
Replaces #1220. Fixes #1214.

To do:

- [X] Replace region/macroregion record for National Capital Region
- [X] Supersede existing regions (polygons) and duplicate locality/localadmin (point) records into new locality (polygon) records
- [x] PIP all records
- [x] PIP descendants

### CODE REVIEW

**Comment by stepps00** [comment]
Noting that https://github.com/whosonfirst-data/whosonfirst-data/commit/bdc6899829d4bfcc0c2dc750ecf44e2e5343fed5 was commited to `master`, rather than this branch. It was reverted via: https://github.com/whosonfirst-data/whosonfirst-data/commit/5dd43d38af6658e71c8333638d85f27c7e15c830.

Records in those commits were added to this PR via: https://github.com/whosonfirst-data/whosonfirst-data/pull/1267/commits/6ecdc12be84ef8bf9f12eeab062d8833093fb76d.

**Comment by noizde** [comment]
Awesome, thank you for taking care of this!

File: data/115/939/726/9/1159397269.geojson [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        "wd:id":"Q13580"
+    },
+    "wof:concordances:hasc:id":"",
+    "wof:country":"PH",
+    "wof:geomhash":"257b6b3a8729506b74ea0d6355292c14",
+    "wof:hierarchy":[
+        {
+            "continent_id":102191569,
+            "country_id":85632509,
+            "locality_id":1159397269,
```
Comment (nvkelso) [inline data/115/939/726/9/1159397269.geojson]: Why a missing `county` here for Metro Manila? Should it be coterminous with itself or...?

File: data/115/939/727/5/1159397275.geojson [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    "name:bcl_x_preferred":[
+        "Rehiyon Nasyunal na Kapitolyo"
+    ],
+    "name:bel_x_preferred":[
+        "\u0421\u0442\u0430\u043b\u0456\u0447\u043d\u044b \u0420\u044d\u0433\u0456\u0451\u043d"
+    ],
+    "name:ben_x_preferred":[
+        "\u09ae\u09c7\u099f\u09cd\u09b0\u09cb \u09ae\u09be\u09a8\u09bf\u09b2\u09be"
+    ],
+    "name:bre_x_preferred":[
```
Comment (nvkelso) [inline data/115/939/727/5/1159397275.geojson]: These look to be name duplicates between many Metro Manila localities. Please make sure only the Manila record has those translations (and remove on the others).

What's the root cause of that?

File: data/115/939/727/7/1159397277.geojson [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    "ne:adm1_cod_1":"PHL-5559",
+    "ne:adm1_code":"PHL-5559",
+    "ne:admin":"Philippines",
+    "ne:area_sqkm":"0.00000000000",
+    "ne:check_me":"20",
+    "ne:code_hasc":"PH.",
+    "ne:code_local":"",
+    "ne:datarank":"5",
+    "ne:diss_me":"5559",
+    "ne:featurecla":"Admin-1 scale rank",
```
Comment (nvkelso) [inline data/115/939/727/7/1159397277.geojson]: This looks all matter of wrong – WOF localities shouldn't map to NE admin-1 (regions).

File: data/115/939/728/3/1159397283.geojson [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    "wd:wordcount":3658,
+    "wof:belongsto":[
+        102191569,
+        85632509,
+        1159397243
+    ],
+    "wof:breaches":[],
+    "wof:concordances":{
+        "fips:code":"RPD9",
+        "gn:id":-99,
```
Comment (nvkelso) [inline data/115/939/728/3/1159397283.geojson]: This isn't a valid GN concordance, -99 elsewhere meant unknown, but in WOF we leave it out to indicate unknown.

File: data/421/168/541/421168541.geojson [hunk 5]
```diff
@@ -218,20 +219,21 @@
...
+            "region_id":1159397243
         }
     ],
     "wof:id":421168541,
-    "wof:lastmodified":1514496440,
+    "wof:lastmodified":1532556659,
     "wof:name":"Makati City",
-    "wof:parent_id":1108697351,
+    "wof:parent_id":1159397243,
     "wof:placetype":"locality",
```
Comment (nvkelso) [inline data/421/168/541/421168541.geojson]: If this is in metro manila and is effectively a `county`, then we should have placetype_alt of `county` listed for this and sister Manila localities.

File: data/115/939/725/7/1159397257.geojson [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+        "hil",
+        "war",
+        "pag",
+        "mdh",
+        "tsg"
+    ],
+    "wof:lastmodified":1532556649,
+    "wof:name":"Makati",
+    "wof:parent_id":1159397243,
+    "wof:placetype":"locality",
```
Comment (nvkelso) [inline data/115/939/725/7/1159397257.geojson]: See placetype_alt of `county` for metro Manila localities comment elsewhere.

**Comment by nvkelso** [comment]
New:

- 1 `region`
- 17 `locality` (with placetype_alt of `county`)

Old:

- 1 `county`
- 17 `region`
- 15 `locality`
- 2 `localadmin` (deprecated)

File: data/115/939/727/5/1159397275.geojson (hunk 2 above)
Comment (stepps00) [reply data/115/939/727/5/1159397275.geojson]: I'm removing the bunk names in an upcoming commit, but the root cause is a shared WikiData concordance ("Q13580"). This concordance exists on all records with the "Metro Manila" (and related) name translations. 

To be safe, I'm going to verify all concordances on the new records too.. :man_shrugging:

File: data/115/939/728/3/1159397283.geojson (hunk 4 above)
Comment (stepps00) [reply data/115/939/728/3/1159397283.geojson]: geonames concordances updated in: e7876c097df1c

File: data/115/939/727/7/1159397277.geojson (hunk 3 above)
Comment (stepps00) [reply data/115/939/727/7/1159397277.geojson]: After reviewing, I think these `ne:` properties are actually correct.. the names match, as do the placetypes.. example in this record:

`"ne:type_en":"Independent Municipality",`

the wikipedia links, names, admin data in the `ne:` also seem to match.

**Comment by stepps00** [comment]
The most recent set of commits address all comments, aside from the `ne:` properties. I've kept them in the records because I suspect they are actually on the correct records.

I've also scrubbed concordances, added population data, and stripped out bunk names.

back to @nvkelso.

**Comment by stepps00** [comment]
Thanks for all the help on this one, @noizde

**Comment by noizde** [comment]
And thank you! Looking forward to seeing the records line up nicely on the Spelunker. :smile:

**Pull request merged**